features/step_definitions/cms/sidebar_steps.rb

```diff
@@ -5,8 +5,6 @@
 end
 
 When /^I switch to (builtin|3scale) content$/ do |group|
-  ensure_javascript
```

Comment:
why is this removed?

Reply:
I think it's nice to stay so that we know this is expected to fail without js. Otherwise it may require more cycles to figure out.